sdk/storage/inc/common/crypt.hpp

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <string>
+
+namespace Azure { namespace Storage {
+
+  std::string HMAC_SHA256(const std::string& text, const std::string& key);
+  std::string Base64Encode(const std::string& text);
+  std::string Base64Decode(const std::string& text);
+  std::string MD5(const std::string& text);
+  std::string CRC64(const std::string& text);
```

Comment:
>CRC64 [](start = 14, length = 5)

Question: How would the interface be when stream is supported in the future?

Reply:
i didn't implement MD5 and CRC64 yet. The interface will be changed.

Reply:
there will be a context